src/tests/ftest/util/job_manager_utils.py

```diff
@@ -126,6 +121,38 @@ def assign_environment_default(self, env_vars):
         """
         pass
 
+    def get_subprocess_state(self, message=None):
+        """Display the state of the subprocess.
+
+        Args:
+            message (str, optional): additional text to include in output.
+                Defaults to None.
+
+        Returns:
+            list: a list of states for the process found. Multiple states should
+                indicate that the job manager and at least one remote process
+                are still running.
+
+        """
+        # Get/display the state of the job manager process
+        state = super(JobManager, self).get_subprocess_state(message)
+        if self._process is not None and self._hosts:
```

Comment:
Very minor suggestion: add comment here "Get/display the state of the remote processes"